demos/mlsys_2020/demo_source.py

```diff
@@ -35,7 +53,7 @@ def draw_grid(dims, vals, grid_size=10, color='#ffffff'):
                         stroke_width=grid_size,  # outline size
                         stroke='black',  # outline color
                         fill=color))  # fill color
-                string_output = str(vals[x][y])
+                string_output = str(vals[dims[1] - 1 - y][x])
```

Comment:
@flaub sorry, wanted to push one more change...

Remember how the matmul results were looking a little weird yesterday? I tracked it down to the `draw_grid` function drawing the arrays weirdly (because the origin of the drawings denote the bottom left, whereas for matrices, the `array[0][0]` element is traditionally placed at the top left).

For example, the array `[[1, 2, 3], [4, 5, 6], [7, 8, 9]]` was showing up as this:
![image](https://user-images.githubusercontent.com/50588693/73976526-5a68a080-48dd-11ea-8da4-171f4d02ca36.png)

I fixed this now:

![image](https://user-images.githubusercontent.com/50588693/73976729-c1865500-48dd-11ea-8396-7ce400beab5a.png)

Reply:
Good catch. Might be good to add a legend to the top (or bottom) and left (or right) edge. Also, might be good to say whether things are major-column or major-row layout.